site/docs/master/migration-case.md

```diff
@@ -2,7 +2,7 @@
 
 *Using Backups and Restores*
 
-Velero can help you port your resources from one cluster to another, as long as you point each Velero instance to the same cloud object storage location. In this scenario, we are also assuming that your clusters are hosted by the same cloud provider. **Note that Velero does not support the migration of persistent volumes across cloud providers.**
+Velero can help you port your resources from one cluster to another, as long as you point each Velero instance to the same cloud object storage location. In this scenario, we are also assuming that your clusters are hosted by the same cloud provider. **Note that Velero does not support the migration of persistent volumes snapshots across cloud providers.** If you would like to migrate volume data between cloud platforms, please use [restic][2], which will backup volume contents at the filesystem level.
```

Comment:
To make it extra explicit, I sugest:

s/Note that Velero does not support/Note that Velero does not natively support

and

s/please use [restic][2], which will backup volume contents/please enable [restic][2], which will allow backup of volume contents

Reply:
Updated!